aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -121,6 +122,23 @@ Tensor& add_relu_(Tensor& self, const Tensor& other, Scalar alpha) {
   return add_relu_impl(self, self, other, alpha);
 }
 
+Tensor& copysign_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(result, self, other);
+  copysign_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor copysign(Tensor& self, const Tensor& other) {
```

Comment:
These should both be `const Tensor&`; self and other are not being modified in place.

Reply:
You are right!!!!